packages/frontend/@n8n/i18n/src/locales/en.json

```diff
@@ -2864,6 +2864,11 @@
 	"dataStore.addColumn.nameInput.placeholder": "Enter column name",
 	"dataStore.addColumn.typeInput.label": "@:_reusableBaseText.type",
 	"dataStore.addColumn.error": "Error adding column",
+	"dataStore.moveColumn.error": "Error moving column",
+	"dataStore.deleteColumn.error": "Error deleting column",
+	"dataStore.columnActions.delete": "Delete column",
+	"dataStore.deleteColumn.confirm.title": "Delete column",
```

Comment:
These two (`dataStore.columnActions.delete` and `dataStore.deleteColumn.confirm.title`) have the same value. I would suggest merging them.